code/go/0chain.net/chaincore/chain/protocol_view_change_integration_tests.go

```diff
@@ -63,6 +76,12 @@ func (c *Chain) SetupSC(ctx context.Context) {
 
 				if txn != nil && c.ConfirmTransaction(ctx, txn, 0) {
 					logging.Logger.Debug("Register node transaction confirmed")
+					func() {
+						defer func() {
+							recover()
+						}()
+						close(doneC)
```

Comment:
same here, I suggest to use 2 different channels to avoid double closing and process them in select to get close at first out of two